### PR TITLE
MINOR: Fixed compiler warnings in LogManagerTest

### PR DESCRIPTION
**Pull request by abbccdda**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by hachikuji — APPROVED** [review]
LGTM

**Comment by ijuma** [comment]
How can this be a checkstyle issue if it's in a Scala file?

**Comment by hachikuji** [comment]
The change itself seems fine, but yeah, we should probably clarify that this is not really a "checkstyle" problem.

**Comment by abbccdda** [comment]
@ijuma @hachikuji correct, I mistake it for another error, let me rephrase the PR title.

**Comment by hachikuji** [comment]
@abbccdda I'd suggest a title like "Fixed compiler warnings in LogManagerTest" or something like that. We don't currently run checkstyle against the scala code.

**Comment by abbccdda** [comment]
@hachikuji sg!
